### ISSUE TITLE
App needs updating for Trakt OAuth flow

### ISSUE THREAD

**Issue by squarefrog** [squarefrog/TraktTopTen#18]
Seriously Trakt - you have to authenticate to just fetch a list of top movies?

Well that's the nail in the coffin for this project then!

**Comment by alberto-h** [comment]
Could you explain what's happening with OAuth? Is this why nothing shows up?

**Comment by squarefrog** [comment]
Indeed. So Trakt have changed their authentication method. Previously anyone used to be able to just get a app key in order to use their API, but they have since changed it. According to [this thread](http://support.trakt.tv/forums/188762-general/suggestions/6925772-where-can-i-get-my-api-key) it seems as though you can just use your Trakt username as the key instead, so might be worth giving that a go!
